bin/reth/src/cli/ext.rs

```diff
@@ -208,7 +208,7 @@ impl<Conf: RethNodeCommandConfig> RethCliExt for NoArgsCliExt<Conf> {
 /// additional CLI arguments.
 ///
 /// Note: This type must be manually filled with a [RethNodeCommandConfig] manually before executing
-/// the [NodeCommand](crate::commands::node::NodeCommand).
+/// the [NodeCommand](https://github.com/paradigmxyz/reth/blob/main/bin/reth/src/commands/node/mod.rs).
```

Comment:
should probably keep them relative to a crate so our doc tests can pick up on any missing links